pkg/runner/builtin/runner.go

```diff
@@ -213,45 +161,49 @@ func (r *Runner) Run(w *queue.Work) error {
 		return err
 	}
 
-	// loop through and execute the notifications and
-	// the destroy all containers afterward.
-	for i, job := range w.Build.Jobs {
-		work := &work{
-			Repo:    w.Repo,
-			Build:   w.Build,
-			Keys:    w.Keys,
-			Netrc:   w.Netrc,
-			Yaml:    w.Yaml,
-			Job:     job,
-			Env:     w.Env,
-			Plugins: w.Plugins,
-		}
-		in, err := json.Marshal(work)
-		if err != nil {
-			return err
-		}
-		workers[i].Notify(in)
-		break
```

Comment:
tracing through the changes. how are notifications being triggered now? Also, how are they being triggered after all jobs are complete, and the build object has been updated?